tests/unit/test_cli.py

```diff
@@ -218,6 +219,13 @@ def test_ensure_assessment_run(ws):
     ws.jobs.list_runs.assert_called_once()
 
 
+def test_ensure_assessment_run_collection(ws):
+    acc_client = create_autospec(AccountClient)
```

Comment:
don't we have a fixture for AccountClient ?

Reply:
updated to use the fixture